src/corelibs/U2Core/src/globals/GUrl.cpp

```diff
@@ -58,6 +58,11 @@ static QString makeFilePathCanonical(const QString& originalUrl) {
         prefix = ":";
         result = result.mid(1);
     } else {
+        // https://learn.microsoft.com/en-us/dotnet/standard/io/file-path-formats#:~:text=%5C%5C.%5CC%3A%5CTest%5CFoo.txt%20%5C%5C%3F%5CC%3A%5CTest%5CFoo.txt
+        // Paths starting with "\\.\" and "\\?\" appear to be the same, but Qt treats them differently.
+        if (result.startsWith("\\\\?\\")) {
```

Comment:
this code should be called only on Windows, right?

Reply:
It would be very strange if such a path came here on Unix. I don't know what this function will return on Unix with or without my changes. However, I can add a check on Windows here :)